### PR TITLE
Add migration for making URLs in DB relative.

### PR DESCRIPTION
**Pull request by rjacobs31**
In question/answer content for `TestingQuestion`, replace replace as recommended by Milton:
>A naive approach would be to do this in a management command
```t.question_content = t.question_content.replace('http://www.dig-it.me/media/', '/media/')```
>
>But a better approach would be to use regex to match the absolute URL `http://*/media/` and replace it with `/media/` without specifying the domain so that the same script can be used in QA and Prod

### CODE REVIEW

File: content/migrations/0035_relative_urls.py [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.db.models import Q
+import re
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
```
Comment (rjacobs31) [inline content/migrations/0035_relative_urls.py:12]: The only part that actually does anything.

File: content/migrations/0035_relative_urls.py [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+        # and orm['appname.ModelName'] for models in other applications.
+        regex = re.compile("http://.*/media/")
+        for item in orm.TestingQuestion.objects.filter(
+                Q(question_content__contains='/media/') | Q(answer_content__contains='/media/')):
+            item.question_content = regex.sub("/media/", item.question_content)
+            item.answer_content = regex.sub("/media/", item.answer_content)
+            item.save()
+
+    def backwards(self, orm):
+        "Write your backwards methods here."
```
Comment (dbreedt) [inline content/migrations/0035_relative_urls.py:25]: Should you not restore this to it's original state if we decide to roll it back?

Comment (rjacobs31) [reply content/migrations/0035_relative_urls.py:25]: The intent behind this migration is to have it run unchanged on both QA and production. As a result we don't really have enough information to reconstruct the full URLs.

**Comment by dbreedt** [comment]
ok understood. :+1:

**Pull request merged**